src/main/java/com/faforever/client/io/ChecksumMismatchException.java

```diff
@@ -0,0 +1,17 @@
+package com.faforever.client.io;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public class ChecksumMismatchException extends Exception {
+  private final Object item;
+  private final String expected;
+  private final String actual;
```

Comment:
Also sorry just realized they are all private final 
```suggestion
@Value
public class ChecksumMismatchException extends Exception {
  Object item;
  String expected;
  String actual;
```